src/features/user/Account/History.tsx

```diff
@@ -64,9 +64,13 @@ export default function History({
   const adSpaceLanguage = i18n.language === 'de' ? 'de' : 'en';
 
   return (
-    <div className="profilePage">
-      <div className={'profilePageTitle'}>{t('me:payments')}</div>
-      <div className={'profilePageSubTitle'}>{t('me:donationsSubTitle')}</div>
+    <div className={'profilePage'}>
+      <div className={`profilePageTitle ${styles.historyTitle}`}>
+        {t('me:payments')}
+      </div>
+      <div className={`profilePageSubTitle ${styles.historySubtitle}`}>
+        {t('me:donationsSubTitle')}
+      </div>
```

Comment:
@prachigarg19 I assume you've not used a `DashboardView` as we have filters alongside. Is there a way to adapt `DashboardView` to allow a 2nd column if needed? Note: this will not apply in specific scenarios like `RegisterTrees` where there is an underlying map element.

Reply:
Resolved in #1853